msp/msp_exp_handler.c

```diff
@@ -726,7 +735,7 @@ void msp_exprecv_syscommand(unsigned char opcode)
 void Timer1_IRQHandler(void)
 {
 	citiroc_daq_set_hvps_temp(hvps_get_temp());
-	// citiroc_daq_set_citi_temp(hkadc_read(CITI_TEMP_CHAN));
+    citiroc_daq_set_citi_temp(hk_adc_get_avg_citi_temp());
```

Comment:
Please use a TAB character here.

Reply:
I could not see this in my Eclipse IDE. I changed the settings and now I can see tabs and space characters. Made the change.